src/tools/changeReportStatus.js

```diff
@@ -20,9 +21,12 @@ export default async function changeReportStatus(ids, status) {
     if (report) {
       auditLogger.info(`Changing status of report: ${id} to ${status}`);
       promises.push(
-        report.update({
-          submissionStatus: status,
-        }),
+        await sequelize.transaction(async (transaction) => report.update(
+          {
+            submissionStatus: status,
+          },
+          transaction,
+        )),
```

Comment:
I don't think we want the `await` in front of the `sequelize.transaction` line because the transaction is added to the `promises` array that eventually passed into `Promise.all`, which will wait for all promises in the array to finish before resolving.

Reply:
I actually think the argument can be made to push the creation of the transaction outside the `for await (const id of idsArray)` on line 18 so all report update calls happen in the same transaction.